File: Soal A/Product.java
public class Product {

    private int price = 0;
    private String idProduct = "";
    // constructor
    public Product(int price, String idProduct) {
        this.price = price;
        this.idProduct = idProduct;
    }

    public Product() {

    }

    public void setPrice(int price) {
        this.price = price;        
    }
    public int getPrice() {
        return this.price;        
    }

    public void setIdProduct(String idPString) {
        this.idProduct = idProduct;
    }
    public String getIdProduct() {
        return this.idProduct;
    }
}

File: Python/Product.py
class Product:

    # constructor
    def __init__(self):
        self.idProduct = "blank"
        self.price = 0

    # methods for id product attribute
    def setIDProduct(self, idProduct):
        self.idProduct = idProduct

    def getIDProduct(self):
        return self.idProduct 

    # methods for price attribute
    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price 
